web3-dev/smart-contracts/sc-execution.rst

```diff
@@ -0,0 +1,68 @@
+========================
+Smart contract execution
+========================
+
+Introduction
+============
+
+In Massa you can execute smart contracts not only deploy and call.
+There is different possibilities to execute smart contracts in different contexts.
+This documentation will explain you the different possibilities and how to use them.
+You can choose the one that match the most your usecase.
+
+All of this way to execute smart contracts are available
```

Comment:
Not understandable as it is.

Reply:
I have written it again. Tell me if it's better